dbaas_controller.go

```diff
@@ -16,10 +16,3 @@
 
 // Package dbaascontroller is a dummy package.
 package dbaascontroller
-
-import (
-	"embed"
-)
-
-//go:embed deploy/*
```

Comment:
why do we need this file now, if we have removed the build directive?

Reply:
I brought it back, to avoid conflicts as Nurlan suggested.
But anyway there si dbaas_controller_test.go, go refuses to run tests for packages that don't have non-test code. That is why there was this dummy file.